packages/twenty-ui/src/theme/constants/FontCommon.ts

```diff
@@ -10,6 +10,7 @@ export const FONT_COMMON = {
   },
   weight: {
     regular: 400,
+    mediumForNavigation: 500,
     medium: 'var(--twentycrm-border-radius-sm)',
```

Comment:
```suggestion
    medium: 'var(--twentycrm-font-weight-medium)',
```

Reply:
Thanks for this suggestion. I thought this was intentional and thus didn't modify the existing code.

Did the correction as asked. Please check now.